File: CodeForces/Python/CF_734A_AntonAndDanik.py
# Problem: 734A. Anton and Danik
# Link: https://codeforces.com/problemset/problem/734/A

n = input()
games = input()

anton_score = games.count('A')
danik_score = games.count('D')

if anton_score > danik_score:
    print("Anton")
elif danik_score > anton_score:
    print("Danik")
else:
    print("Friendship")

File: CodeForces/Java/CF_734A_AntonAndDanik.java
//  Problem: 734A. Anton and Danik
//  Link: https://codeforces.com/problemset/problem/734/A

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner scan = new Scanner(System.in);
    
    int n = scan.nextInt();
    String games = scan.next();
    scan.close();

    int score = 0;
    
    for (int i = 0; i < n; i++) {
        if (games.charAt(i) == 'A') {
            score++;
        } else {
            score--;
        }
    }
    
    if (score > 0) {
        System.out.println("Anton");
    } else if (score < 0) {
        System.out.println("Danik");
    } else {
        System.out.println("Friendship");
    }
  }
}
